eclass/coreos-kernel.eclass

```diff
@@ -93,10 +93,10 @@ kmake() {
 	if gcc-specs-pie; then
 		kernel_cflags="-nopie -fstack-check=no"
 	fi
-	emake -C "${KERNEL_DIR}" \
+	emake \
 		ARCH="${kernel_arch}" \
 		CROSS_COMPILE="${CHOST}-" \
-		KBUILD_OUTPUT="${S}" \
+		KBUILD_OUTPUT="build" \
```

Comment:
This looks like it should be "${S}/build", but I'm not familiar with any of this machinery.